solidity/ecdsa/slither.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "detectors_to_exclude": "assembly,timestamp,solc-version,naming-convention",
+  "hardhat_artifacts_directory": "./build",
+  "filter_paths": "test|node_modules|BytesLib.sol|ModUtils.sol|AltBn128.sol"
```

Comment:
I think these guys BytesLib.sol|ModUtils.sol|AltBn128.sol don't belong here since they were created in random-beacon

Reply:
It's likely we will copy-paste those libraries from random beacon. But, yeah, we can remove this for now.

Reply:
056bfdf826848596f7e98a1e9d955828a1925fc1